### ISSUE TITLE
Add XMLBear in .coafile

### ISSUE THREAD

**Issue by co-robo** [coala/coala-eclipse#39]
Add XMLBear to repository and fix errors raised by coala.
label: difficulty/low
Opened by @aashutoshrathi at [Gitter](https://gitter.im/coala/coala?at=59f98ca25a1758ed0f75523b)

**Comment by prashant0598** [comment]
@Makman2 I would like to work on this issue. how should i proceed?

**Comment by Makman2** [comment]
1. the issue assigned, however: @Lucifer24 you working o nthis issue?
2. @prashant0598 please read our newcomer guide ;)

**Comment by prashant0598** [comment]
@Makman2 i would like to work on this issue

**Comment by Lucifer24** [comment]
@Makman2 Yes I am working. I ran into some trouble with my laptop thats why was unable to do it.
Sorry for the delay.

**Comment by aashutoshrathi** [comment]
@Makman2 What is the point of rushing for this PR, as all PRs will be blocked till Travis isn't fixed right?

**Comment by Makman2** [comment]
@aashutoshrathi I'm not rushing, just confirming if it's worked on. If not, the issue can be reassigned (even if ~~Travis~~ rultor is not working).

**Comment by Mohit17067** [comment]
Can I work on this if no one's working?

**Comment by aashutoshrathi** [comment]
Yes, I think you can as this is stale from 2 months.

**Comment by Makman2** [comment]
Reassigning due to inactivity.
